Severity (#16921)
carbon-design-system/carbon: docs/guides/support.md

| :-------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------------------: | ---------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | https://github.com/carbon-design-system/carbon/labels/severity%3A%201 | Affects major functionality. Blocks users from completing tasks, exposes intellectual property or damages the brand. No workaround. | The kit is has major functionality defects, designers can’t design and there is no work-around                   | The documentation damages the brand or exposes intellectual property.                                                                          | The code has major functionality defects, breaking builds and can’t be worked around.                            | If the problem is in production, it must be fixed as soon as possible in a patch.                                                                                                |

Reviewer: ```suggestion
| https://github.com/carbon-design-system/carbon/labels/severity%3A%201 | Affects major functionality. Blocks users from completing tasks, exposes intellectual property or damages the brand. No workaround. | The kit is has major functionality defects, designers can’t design and there is no workaround                   | The documentation damages the brand or exposes intellectual property.                                                                          | The code has major functionality defects, breaking builds and can’t be worked around.                            | If the problem is in production, it must be fixed as soon as possible in a patch.                                                                                                |
```

> | [1](https://github.com/carbon-design-system/carbon/labels/severity%3A%201) | Affects major functionality. Blocks users from completing tasks, exposes intellectual property or damages the brand. No workaround. | The kit is has major functionality defects, designers can’t design and there is no work-around                   | The documentation damages the brand or exposes intellectual property.                                                                          | The code has major functionality defects, breaking builds and can’t be worked around.                            | If the problem is in production, it must be fixed as soon as possible in a patch.                                                                                                |

Reviewer 2: If you remove the markdown formatting around these links GitHub will display the actual label itself. It's still a link, too.

```suggestion
| https://github.com/carbon-design-system/carbon/labels/severity%3A%201 | Affects major functionality. Blocks users from completing tasks, exposes intellectual property or damages the brand. No workaround. | The kit is has major functionality defects, designers can’t design and there is no work-around                   | The documentation damages the brand or exposes intellectual property.                                                                          | The code has major functionality defects, breaking builds and can’t be worked around.                            | If the problem is in production, it must be fixed as soon as possible in a patch.                                                                                                |
```

> | [2](https://github.com/carbon-design-system/carbon/labels/severity%3A%202) |                     Affects major functionality. Does not block users from completing tasks. Has a workaround.                      | The kit has a major functionality defect, but it doesn't block users from completing tasks. It has a workaround. | The documentation is wrong or missing major details. It conflicts with what we are saying in another part of the system.                       | The code has a major functionality defect, but it doesn't block users from completing tasks. It has a workaround | A release will be moved into Production with defects at this severity level included. These defects will have priority in future sprints to be resolved ASAP.                    |

Reviewer 2: ```suggestion
| https://github.com/carbon-design-system/carbon/labels/severity%3A%202 |                     Affects major functionality. Does not block users from completing tasks. Has a workaround.                      | The kit has a major functionality defect, but it doesn't block users from completing tasks. It has a workaround. | The documentation is wrong or missing major details. It conflicts with what we are saying in another part of the system.                       | The code has a major functionality defect, but it doesn't block users from completing tasks. It has a workaround | A release will be moved into Production with defects at this severity level included. These defects will have priority in future sprints to be resolved ASAP.                    |
```

> | https://github.com/carbon-design-system/carbon/labels/severity%3A%203 |                     Affects minor functionality. Does not block users from completing tasks. Has a workaround.                      | The kit has a minor functionality defect. It doesn't block users from completing tasks. Has a work around.       | The documentation is mis-leading, somewhat true, or missing minor details. It doesn't present conflicts with other parts of the design system. | The code has a minor functionality defect. It doesn't block users from completing tasks. Has a work around.      | A release will be moved into Production with defects at this severity level included. These defects will have priority in future sprints after sev2's.                           |

Reviewer: ```suggestion
| https://github.com/carbon-design-system/carbon/labels/severity%3A%203 |                     Affects minor functionality. Does not block users from completing tasks. Has a workaround.                      | The kit has a minor functionality defect. It doesn't block users from completing tasks. Has a workaround.       | The documentation is mis-leading, somewhat true, or missing minor details. It doesn't present conflicts with other parts of the design system. | The code has a minor functionality defect. It doesn’t block users from completing tasks. Has a workaround.      | A release will be moved into Production with defects at this severity level included. These defects will have priority in future sprints after sev2’s.                           |
```

> | [3](https://github.com/carbon-design-system/carbon/labels/severity%3A%203) |                     Affects minor functionality. Does not block users from completing tasks. Has a workaround.                      | The kit has a minor functionality defect. It doesn't block users from completing tasks. Has a work around.       | The documentation is mis-leading, somewhat true, or missing minor details. It doesn't present conflicts with other parts of the design system. | The code has a minor functionality defect. It doesn't block users from completing tasks. Has a work around.      | A release will be moved into Production with defects at this severity level included. These defects will have priority in future sprints after sev2's.                           |

Reviewer 2: ```suggestion
| https://github.com/carbon-design-system/carbon/labels/severity%3A%203 |                     Affects minor functionality. Does not block users from completing tasks. Has a workaround.                      | The kit has a minor functionality defect. It doesn't block users from completing tasks. Has a work around.       | The documentation is mis-leading, somewhat true, or missing minor details. It doesn't present conflicts with other parts of the design system. | The code has a minor functionality defect. It doesn't block users from completing tasks. Has a work around.      | A release will be moved into Production with defects at this severity level included. These defects will have priority in future sprints after sev2's.                           |
```

> | [4](https://github.com/carbon-design-system/carbon/labels/severity%3A%204) |                                        Unrelated to a user task. Does not need a workaround.                                        | Unrelated to a user task. Does not need a workaround.                                                            | Unrelated to a user task. Does not need a workaround.                                                                                          | Unrelated to a user task. Does not need a workaround.                                                            | A release will be moved into Production with defects at this severity level included. These will often not be addressed by the maintainer team as a priority in future releases. |

Reviewer 2: ```suggestion
| https://github.com/carbon-design-system/carbon/labels/severity%3A%204 |                                        Unrelated to a user task. Does not need a workaround.                                        | Unrelated to a user task. Does not need a workaround.                                                            | Unrelated to a user task. Does not need a workaround.                                                                                          | Unrelated to a user task. Does not need a workaround.                                                            | A release will be moved into Production with defects at this severity level included. These will often not be addressed by the maintainer team as a priority in future releases. |
```
